data_extractor/google_search_history/__init__.py

```diff
@@ -0,0 +1,55 @@
+""" Script to extract info from Google Browser History """
+
+__version__ = '0.1.0'
+
+import json
+import datetime as DT
+import numpy as np
+import pandas as pd
+import re
+
+
+def process(data):
+    """Return relevant data from browser history pre and post specific date
+    Args:
+        data: BrowserHistory.json file
+    Returns:
+        dict: dict with summary and DataFrame with extracted data
+    """
+    # Enter date of event X (in this case 'avondklok')
+    date = {'ingang_avondklok': np.datetime64(DT.datetime(2021, 1, 23, 21)).view('<i8'),
+            'einde_avondklok': np.datetime64(DT.datetime(2021, 4, 28, 4, 30)).view('<i8')}
+    # Enter news sites
+    newssites = 'news.google.com|nieuws.nl|nos.nl|www.rtlnieuws.nl|nu.nl|at5.nl|ad.nl|bd.nl|telegraaf.nl|volkskrant.nl' \
+        '|parool.nl|metronieuws.nl|nd.nl|nrc.nl|rd.nl|trouw.nl'
+    # Extract moment (pre/during/after event X) and website (news/other)
+    results = {'Moment': [], 'Website': []}
+    for data_unit in data["Browser History"]:
```

Comment:
This counts all occurrences of a website in the json, i.e. all type of `page_transition`, thus also `RELOAD`. Not sure if this is desirable. Check with researcher.

Reply:
Added filter to exclude 'RELOAD' page_transitions.